jetty-core/jetty-http-spi/src/main/java/org/eclipse/jetty/http/spi/JettyHttpServer.java

```diff
@@ -48,18 +51,28 @@ public class JettyHttpServer extends com.sun.net.httpserver.HttpServer
     private final Map<String, JettyHttpContext> _contexts = new HashMap<>();
     private final Map<String, Connector> _connectors = new HashMap<>();
     private InetSocketAddress _addr;
+    private int _backlog;
+    private HttpsConfigurator configurator;
 
-    public JettyHttpServer(Server server, boolean shared)
+    public JettyHttpServer(Server server, boolean shared, InetSocketAddress addr, int backlog)
```

Comment:
Thanks for the contribution.
This is breaking backward. Would it be possible to keep backward compatibility? And add `@deprecated` where needed. Eventually remove when merging in 12.1.

Reply:
resolved